spec/controllers/case_workers/admin/allocations_controller_spec.rb

```diff
@@ -91,8 +91,8 @@
     before(:each) do
       expect(CaseWorkerService).to receive(:new).and_return(case_worker_service_instance)
       expect(Claims::CaseWorkerClaims).to receive(:new).and_return(case_worker_claims_instance)
-      expect(Allocation).to receive(:new).with(expected_params)
-      expect(Allocation).to receive(:new).with(expected_params_with_user).and_return(allocation)
+      expect(Allocation).to receive(:new).with(ActionController::Parameters.new(expected_params).permit!)
```

Comment:
**[not a blocker/nice to have]** 💭 Given we're likely to have quite some definitions of params as the above, having some helper (would be included in the `spec/rails_helper.rb` like the one suggested [here](https://stackoverflow.com/a/45468322) might help DRY things a bit and make the expectations shorter an nicer to read.

Reply:
ooh.  Nice find..